app/assets/stylesheets/views/_answers.scss

```diff
@@ -0,0 +1,9 @@
+.answers {
+  .add-title-margin {
+    @include responsive-top-margin;
+  }
+
+  strong {
```

Comment:
We should use `rich_govspeak: true` if it's needed for the body text. Or it should be removed if bold is bad.